web-app/django/VIM/apps/instruments/views/instrument_list.py

```diff
@@ -238,26 +377,7 @@ def get(self, request, *args, **kwargs):
             request.session["active_language_en"] = language_en
         return super().get(request, *args, **kwargs)
 
-    def get_queryset(self) -> Union[QuerySet[Instrument], list[SolrInstrument]]:
-        language = self.get_active_language()
-        instrumentname_prefetch_manager = Prefetch(
-            "instrumentname_set",
-            queryset=InstrumentName.objects.filter(language=language),
-        )
-        hbs_facet = self.request.GET.get("hbs_facet", None)
-        if hbs_facet:
-            return (
-                Instrument.objects.filter(hornbostel_sachs_class__startswith=hbs_facet)
-                .select_related("thumbnail")
-                .prefetch_related(instrumentname_prefetch_manager)
-            )
-
-        search_query = self.request.GET.get("query", "").strip()
-
-        if search_query:
-            # Return a special marker for Solr search that will be handled in paginate_queryset
-            return SOLR_SEARCH_MARKER
-
-        return Instrument.objects.select_related("thumbnail").prefetch_related(
-            instrumentname_prefetch_manager
-        )
+    def get_queryset(self) -> str:
```

Comment:
You don't need this if you're not working with a Django queryset. (See the above comment from @dchiller  about not using `ListView` if you're not using the Django ORM.)